core/sup/rebar.config

```diff
@@ -1,5 +1,8 @@
+ {deps, [lager,jiffy,getopt,recon,zucchini,erlang_localtime
+,{inet_cidr, "1.0.2", {git, "https://github.com/2600hz/erlang-inet_cidr.git", {tag, "1.0.2"}}}
+]}.
 {lib_dirs, ["../../deps", "../../core"]}.
-{escript_incl_apps, [getopt, kazoo, kazoo_stdlib, kazoo_config, zucchini, lager]}.
```

Comment:
`sup` is a command line interface for communicating with Kazoo, while getopt is a tool for parsing command line options.  Without it, sup will not work.

Reply:
While resolving sup build issue we moved this to deps.
Now I tried adding back and build is working fine.